### PR TITLE
Added a new entry in Important section for 1.11

### PR DESCRIPTION
**Pull request by anithamuthu79**
Based on recent customer issues on overlay networks failure, Jena reported the following information to be documented in the "Important" section of https://docs.mesosphere.com/1.11/installing/ent/upgrading/
and
https://docs.mesosphere.com/1.11/installing/ent/custom/advanced/#install-dcos

- Due to cluster configuration issues that caused overlay networks to fail in DC/OS 1.11.0, 1.11.1, and 1.11.2, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster. If you have already upgraded to DC/OS 1.11.x without configuring `enable_ipv6` or if `config.yaml` file is set to `true` then do not add new nodes until DC/OS 1.11.3 has been released. You can find additional information and a more robust remediation procedure in our latest critical product advisory.

## Urgency
- [ ] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Add [redirects](https://github.com/mesosphere/dcos-docs-site/wiki/Redirects).
- Change all affected versions (e.g. 1.7, 1.8, 1.9, 1.10, 1.11).
- See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/wiki/Contributing).

### CODE REVIEW

File: pages/1.11/installing/ent/custom/advanced/index.md [hunk 1]
```diff
@@ -191,7 +191,8 @@ In this step you create a custom DC/OS build file on your bootstrap node and the
 
 **Important:**
 
-- Do not install DC/OS until you have these items working: ip-detect script, DNS, and NTP everywhere. For help with troubleshooting, see the [documentation](/1.11/installing/ent/troubleshooting/).
+- Due to cluster configuration issues that caused overlay networks to fail in DC/OS 1.11.0, 1.11.1, and 1.11.2, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster. If you have already upgraded to DC/OS 1.11.x without configuring `enable_ipv6` or if `config.yaml` file is set to `true` then do not add new nodes until DC/OS 1.11.3 has been released. You can find additional information and a more robust remediation procedure in our latest critical product advisory.
```
Comment (pavisandhu) [inline pages/1.11/installing/ent/custom/advanced/index.md]: Should we change the first sentence to the following?
"Due to a cluster configuration issue with overlay networks, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster." 
The phrase "issues that caused overlay networks to fail" seems negative and alarming. But if it's important to emphasize the severity of the issue so customers don't miss it, we can keep the sentence as is. @jenafermiller What's your preference?

File: pages/1.11/installing/ent/upgrading/index.md [hunk 2]
```diff
@@ -15,7 +15,8 @@ If this upgrade is performed on a supported OS with all prerequisites fulfilled,
 **Important:**
 
 - Review the [release notes](/1.11/release-notes/) before upgrading DC/OS.
-- There are new options in the `config.yaml` file which must be declared prior to upgrading. Even if you have previously installed DC/OS successfully with your `config.yaml` file, the file will require new additions to function with DC/OS 1.11. Pay specific attention to the `fault_domain_enabled` and `enable_ipv6`. Please review the sample file [here](/latest/installing/ent/custom/advanced/#create-a-configuration-file)
+- Due to cluster configuration issues that caused overlay networks to fail in DC/OS 1.11.0, 1.11.1, and 1.11.2, we currently recommend setting `enable_ipv6` to `false` in `config.yaml` when upgrading _or_ configuring a new cluster. If you have already upgraded to DC/OS 1.11.x without configuring `enable_ipv6` or if `config.yaml` file is set to `true` then do not add new nodes until DC/OS 1.11.3 has been released. You can find additional information and a more robust remediation procedure in our latest critical product advisory.
```
Comment (pavisandhu) [inline pages/1.11/installing/ent/upgrading/index.md]: Should we change the first sentence to the following?
"Due to a cluster configuration issue with overlay networks, we currently recommend setting enable_ipv6 to false in config.yaml when upgrading or configuring a new cluster."
The phrase "issues that caused overlay networks to fail" seems negative and alarming. But if it's important to emphasize the severity of the issue so customers don't miss it, we can keep the sentence as is. @jenafermiller What's your preference?

Comment (jenafermiller) [reply pages/1.11/installing/ent/upgrading/index.md]: I like Pavi's suggestion.  It's serious but I don't want to alarm.  Let's make that change. 👍 I will have a product advisory link to add in a few minutes.

Comment (anithamuthu79) [reply pages/1.11/installing/ent/upgrading/index.md]: SGTM. Please share the link here so we can track our changes/inputs in one place.

**Comment by anithamuthu79** [comment]
@pavisandhu @jenafermiller  I've
screen shot 2018-05-21 at 12 37 00 pm
screen shot 2018-05-21 at 12 36 49 pm

 tested the local build and the changes reflected perfectly. Could you review and approve the changes?

**Pull request merged**